gym/lib/gym/options.rb

```diff
@@ -158,6 +158,10 @@ def self.plain_options
                                      env_name: "GYM_SDK",
                                      description: "The SDK that should be used for building the application",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :toolchain,
+                                     env_name: "GYM_TOOLCHAIN",
+                                     description: "The toolchain that should be used for building the application",
```

Comment:
Is there a good way to validate the input, or to show some of the available values?